app/commands/decvol.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Execute on decvol command
+ * @param {Discord.Message} msg - discord message
+ */
+const decvol = async (msg) => {
+    const { musicQueue } = require("../utils/MusicQueue");
+
+    // check if user in the voice channel
+    const voiceChannel = msg.member.voice.channel;
+    if (!voiceChannel) {
+        return msg.channel.send(
+            `${msg.author.toString()} You need to be in a voice channel to decrease volume!`
+        );
+    }
+    let clientVoiceConnection = msg.guild.voice.connection;
+    if (!clientVoiceConnection) {
+        return msg.channel.send(
+            `❌ ${msg.author.toString()} Bot must be in voice channel`
+        );
+    }
+
+    let percentage = 25 / 100; // if you want to use args just adjust the "25" to argument variable.
+
+    const serverQueue = musicQueue.get(msg.guild.id);
+    let volume = serverQueue.connection.dispatcher.volume - percentage;
```

Comment:
When decreasing the volume, it can go beyond 0%. I think updating this line to this would fix the issue:
```javascript
let volume = Math.max(serverQueue.connection.dispatcher.volume - percentage,0);
```